docs/integration/KafkaIntegration.md

```diff
@@ -276,13 +275,15 @@ Important thing to remember is that Kafka server addresses/schema registry addre
-      
 
-| Name                                              | Importance | Type                       | Default value    | Description                                                                                                                                                                                                 |
-|---------------------------------------------------|------------|----------------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| kafkaEspProperties.defaultMaxOutOfOrdernessMillis | Medium     | duration                   | 60s              | Configuration of [bounded of orderness watermark generator](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/datastream/event-time/built_in/#fixed-amount-of-lateness) used by Kafka sources |
-| consumerGroupNamingStrategy                       | Low        | processId/processId-nodeId | processId-nodeId | How consumer groups for sources should be named                                                                                                                                                             |
-| avroKryoGenericRecordSchemaIdSerialization        | Low        | boolean                    | true             | Should Avro messages from topics registered in schema registry be serialized in optimized way, by serializing only schema id, not the whole schema                                                          |
+
+| Name                                              | Importance | Type                       | Default value      | Description                                                                                                                                                                                                     |
+|---------------------------------------------------|------------|----------------------------|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| kafkaEspProperties.defaultMaxOutOfOrdernessMillis | Medium     | long                       | 60000 (60 seconds) | Configuration of [bounded out of orderness watermark generator](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/datastream/event-time/built_in/#fixed-amount-of-lateness) used by Kafka sources |
+| idlenessConfig.enableIdleTimeout                  | Medium     | boolean                    | true               | Enabling [idleness](https://nightlies.apache.org/flink/flink-docs-stable/docs/connectors/datastream/kafka/#idleness) used by Kafka sources                                                                      |
```

Comment:
One more polishing that we can do. We can rename it to `idleTimeout.enabled` and `idleTimeout.duration`. WDYT?

Reply:
Done :+1: